charts/piraeus/values.yaml

```diff
@@ -18,8 +18,16 @@ operator:
   controllerSet:
     controllerImage: "quay.io/piraeusdatastore/piraeus-server:v1.7.0"
     dbCertSecret: ""
+    sslSecret: ""
+    sslKeystorePassword: "linstor"
+    sslKeyPassword: "linstor"
+    sslTruststorePassword: "linstor"
```

Comment:
You put the SSL keys in an "ssl" subsection in the CRD, which feels neater to me. Why not also do that here?

Reply:
It's recommended by [helm](https://helm.sh/docs/chart_best_practices/values/) because it makes it easier to override "flat" values. However Now thinking about it, in 99% (100%?) of all cases the other `ssl` parameters won't be touched, so it might as well be its own subsection

Reply:
If we decide not to make the passwords configurable at all, then this will become irrelevant anyway.

Reply:
removed, password is always `linstor` now